kubernetes/samples/scripts/terraform/template.tfvars

```diff
@@ -0,0 +1,47 @@
+# Example TF variables file for cluster creation
```

Comment:
missing copyright header

Reply:
most other samples call this file terraform.tfvars.example - that seems to be the convention